microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/internal/graphql/properties/MicroProfileGraphQLProvider.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+* Copyright (c) 2020 IBM Corporation and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     IBM Corporation - initial API and implementation
+*******************************************************************************/
+
+package org.eclipse.lsp4mp.jdt.internal.graphql.properties;
+
+import org.eclipse.lsp4mp.jdt.internal.graphql.MicroProfileGraphQLConstants;
+
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.jdt.core.IJavaProject;
+import org.eclipse.lsp4mp.jdt.core.AbstractStaticPropertiesProvider;
+import org.eclipse.lsp4mp.jdt.core.MicroProfileCorePlugin;
+import org.eclipse.lsp4mp.jdt.core.SearchContext;
+import org.eclipse.lsp4mp.jdt.core.utils.JDTTypeUtils;
+
+/**
+ * Properties provider to collect MicroProfile properties from MicroProfile GraphQL 
```

Comment:
Would change this comment to something like `* Properties provider that provides static MicroProfile GraphQL properties` since this PR uses static properties defined in JSON instead of dynamically collecting them.